src/components/board/Board.vue

```diff
@@ -284,6 +284,7 @@ export default {
 					padding-top: 15px;
 					margin-top: -10px;
 					scrollbar-gutter: stable;
+					max-height: 94%;
```

Comment:
This seems like an odd magic number, can we caluclate that rather somehow based on what elements reduce the height from 100%?

Reply:
I have checked again, we can fix this issue by removing `min-height` of stack header. So, we don't need that `max-height` anymore.

Reply:
All right, makes sense 👍